libraries/validate/src/schemaset.js

```diff
@@ -121,7 +107,14 @@ class SchemaSet {
   async validator(rootUrl) {
     await publish({cfg: this.cfg, schemaset: this, rootUrl});
 
-    const ajv = Ajv({useDefaults: true, format: 'full', verbose: true, allErrors: true});
+    const ajv = Ajv({
+      useDefaults: true,
+      format: 'full',
+      verbose: true,
+      // schema validation occurs in the tests and need not be done here
```

Comment:
Thank you for this comment. I would ask this exact question a year from now otherwise.